test/functional/api/cas/ioclass_config.py

```diff
@@ -21,22 +21,25 @@
 
 MAX_IO_CLASS_ID = 32
 MAX_IO_CLASS_PRIORITY = 255
+DEFAULT_IO_CLASS_ID = 0
+DEFAULT_IO_CLASS_PRIORITY = 22
+DEFAULT_IO_CLASS_RULE = "unclassified"
 MAX_CLASSIFICATION_DELAY = timedelta(seconds=6)
 IO_CLASS_CONFIG_HEADER = "IO class id,IO class name,Eviction priority,Allocation"
 
 
 @functools.total_ordering
 class IoClass:
     def __init__(self, class_id: int, rule: str = '', priority: int = None,
-                 allocation: bool = True):
+                 allocation: str = "1.00"):
```

Comment:
Why don't you use float instead of str?

Reply:
Using string instead of float will allow us to create negative tests with invalid allocation values.